mobile-app/app/components/IconButton.tsx

```diff
@@ -11,28 +12,33 @@ interface IconButtonProps extends TouchableOpacityProps {
   iconType: IconType
   iconSize: number
   iconLabel?: string
+  disabled?: boolean
 }
 
 export function IconButton (props: IconButtonProps): JSX.Element {
+  const {
+    disabled = false
+  } = props
   return (
     <ThemedTouchableOpacity
-      dark={tailwind('border border-gray-400 rounded bg-gray-900')}
-      light={tailwind('border border-gray-300 rounded bg-white')}
+      light={tailwind(classNames({ 'border-gray-300 bg-white': !disabled, 'border-gray-100 bg-gray-100': disabled }))}
```

Comment:
You don't need to call classNames implicitly. It's already called inside tailwind. Check tailwind method

Reply:
Ahh okay, because I was looking at the official documentation of tailwind-rn and tailwindcss package and couldn't find any official documentation that mention this, then i realized it was customized within our project by fuxing.

Updated